middle_end/flambda/un_anf.ml

```diff
@@ -306,6 +316,7 @@ let environment_vars = loop ~depth:0 ~environment_vars:V.Set.empty clam in
       | More_than_one -> (linear, V.Set.add var used, assigned)
       | Assigned -> (linear, V.Set.add var used, V.Set.add var assigned))
       t (V.Set.empty, V.Set.empty, V.Set.empty)
+      acc
```

Comment:
I don't think this addition does anything useful. As far as I can tell from a quick glance, `acc` isn't even defined in this scope.